device.go

```diff
@@ -174,9 +174,20 @@ func getDeviceName(s *sandbox, devID string) (string, error) {
 }
 
 func getPCIDeviceNameImpl(s *sandbox, pciID string) (string, error) {
-	pciAddr, err := getDevicePCIAddress(pciID)
-	if err != nil {
-		return "", err
+	var pciAddr string
+	var err error
+
+	const sizeBDFStr = 12
```

Comment:
It worths mentioning BDF format `0000:00:00.0` here to help understand the constants.